test/e2e/common.go

```diff
@@ -41,6 +51,12 @@ func LogFromFile(logFile string) {
 	Logf(string(data))
 }
 
+// return only the boolean value from ParseBool.
+func getBool(s string) bool {
+	b, _ := strconv.ParseBool(s)
+	return b
```

Comment:
Maybe add an error check just to be sure? Otherwise we could miss empty strings or other unexpected situations.
```suggestion
	b, err := strconv.ParseBool(s)
	Expect(err).To(BeNil())
	return b
```

Reply:
Thanks indeed that was missing